.github/workflows/release-docker.yml

```diff
@@ -90,7 +90,7 @@ jobs:
           password: ${{ secrets.DOCKERHUB_PAT }}
```

Comment:
btw seems like docker hub does not parse notes in correct way

<img width="1084" alt="Image" src="https://github.com/user-attachments/assets/58ada59b-6fb2-4c43-a77e-75f447d1266b" />